.github/actions/templates/avm-validateModuleDeployment/action.yml

```diff
@@ -57,6 +57,43 @@ runs:
         creds: ${{ env.AZURE_CREDENTIALS }}
         enable-AzPSSession: true
 
+    # [Set Deployment Location] task(s)
+    # ---------------------------
+    - name: "Set Deployment Location"
+      id: set-deployment-location
+      uses: azure/powershell@v1
+      with:
+        azPSVersion: "latest"
+        inlineScript: |
+          # Grouping task logs
+          Write-Output '::group::Get Recommended Regions'
+
+          # Load used functions
+          . (Join-Path $env:GITHUB_WORKSPACE 'avm' 'utilities' 'pipelines' 'sharedScripts' 'Get-AzRecommendedRegions.ps1')
+
+          # Set fucntion input parameters
+          $functionInput = @{
+              excludedRegions = @("westeurope","westus2","eastus2","centralus","southcentralus","brazilsouth","koreacentral","qatarcentral")
```

Comment:
Maybe we should rather have it as a default value in the script as opposed to explicitely in the pipeline. As the YAML file has no script highlighting it's a bit hard to see. Also, we should display it in a vertical list as it's easier to read. E.g.,

```PowerShell
$excludedRegions = @(
    'westeurope',
    'westus2',
    'eastus2',
    'centralus',
    'southcentralus',
    'brazilsouth',
    'koreacentral',
    'qatarcentral'
)
```

Reply:
Same for the `knownRegionList` parameter

Reply:
I took this out of defualt values in the script as I thought the pipeline would be the preferred place. I can revert this if we want?

Reply:
Added

Reply:
Just comes down to keeping code out of the YAML. Subjectively, the only reasonable place to host these values would be in a `settings.yml` - but as we don't have it, it would be a subjectively bad practice to have 'magic values' stored in a workflow file. Just inking that troubleshooting could be ... challenging :D